docs/semgrep-ci/ci-environment-variables.md

```diff
@@ -299,7 +299,7 @@ steps:
 
 ### `BITBUCKET_TOKEN`
 
-Set `BITBUCKET_TOKEN` to enable Semgrep to leave PR or MR comments in Bitbucket Cloud. The value of this environment variable must be a Personal Access Token (PAT) generated from Bitbucket Cloud. See [Bitbucket PR comments](/semgrep-appsec-platform/bitbucket-pr-comments) for instructions.
+Set `BITBUCKET_TOKEN` to enable Semgrep to leave PR or MR comments in Bitbucket Cloud. The value of this environment variable must be a Personal Access Token (PAT) generated from Bitbucket Cloud. See [Bitbucket PR comments](/category/bitbucket-pr-comments) for instructions.
```

Comment:
Does the link here need to be `bitbucket-cloud-pr-comments`/`bitbucket-data-center-pr-comments`?

Also, I think this section can be marked as *Optional* since the primary way to set up BBC/BBDC is by connecting the SCM via the method above. Maybe we can keep this section but add a preface like "If you are not receiving PR comments and/or code hyperlinks are not displayed in the app, then try setting the `BITBUCKET_TOKEN` env variable".

Reply:
Got it. I changed the link the the BBC page, and changed the verbiage so that it's optional.